packages/react-hooks/src/hooks/steps.ts

```diff
@@ -0,0 +1,19 @@
+import {useState, useCallback} from 'react';
+
+export function useSteps(steps: number, initialStep = 0) {
+  const [currentStep, setStep] = useState(initialStep);
```

Comment:
super nitpicky suggestion about the useState naming conventions: 
```suggestion
  const [currentStep, setCurrentStep] = useState(initialStep);
```

totally inconsequential here but reads a little clearer :)